### ISSUE TITLE
architecture diagram is missing

### ISSUE THREAD

**Issue by rajputvishwas** [wso2/product-microgateway#1135]
### Description:

I am following read me documentation, where micro gateway architecture is missing. While clicking on alt text gives page not found error

![image missing](https://user-images.githubusercontent.com/15668995/76584613-36166a00-6502-11ea-8efc-77ff51758912.PNG)

**Comment by praminda** [comment]
Could you please update the issue body with correct issue template. https://github.com/wso2/product-microgateway/blob/master/.github/ISSUE_TEMPLATE/documentation_template.md

**Comment by rajputvishwas** [comment]
There is an issue with README.md file, for wso2/product-microgateway. The architecture diagram of microgateway is missing from the file.

**Comment by Rajith90** [comment]
Fixed with #1134
